org-formation/organization.yaml

```diff
@@ -363,6 +364,21 @@ Organization:
         budget-alarm-threshold: 10000
         budget-alarm-threshold-email-recipient: cnb@sagebase.org
 
+  CnbDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-cnb-dev
+      RootEmail: aws-cnb-dev@sagebase.org
+      Alias: org-sagebase-cnb-dev
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: CNB
+        Project: Challenges
+        AccountOwner: thomas.schaffter@sagebionetworks.org
+        CostCenter: NIH-ITCR / 101600
+        budget-alarm-threshold: 10000
+        budget-alarm-threshold-email-recipient: aws-cnb-dev@sagebase.org
```

Comment:
I just checked on Google Groups and it looks like Joni is monitoring budget alarms. Thanks!

Reply:
It shouldn't be an IT member who monitors budget alarms but rather the account owner, in this case someone from "CnB".

Reply:
@tschaffter I'm the current owner of the google group until the AWS account is created and I can set its root password. Once that's done, I'll transfer ownership of the google group to you as the account owner.